runtime/parachains/src/configuration.rs

```diff
@@ -734,18 +735,6 @@ pub mod pallet {
 			})
 		}
 
-		/// Set the maximum number of dispute spam slots.
```

Comment:
Should I remove this setter when `dispute_max_spam_slots` is completely removed?

Reply:
I don't see a reason to keep it.